tokens/dist/css/spectrum/custom-light-vars.css

```diff
@@ -1,12 +1,5 @@
 .spectrum--light,
 .spectrum--lightest{
-  --spectrum-menu-item-background-color-default-rgb:0, 0, 0;
-  --spectrum-menu-item-background-color-default-opacity:0;
-  --spectrum-menu-item-background-color-default:rgba(var(--spectrum-menu-item-background-color-default-rgb), var(--spectrum-menu-item-background-color-default-opacity)); /* --spectrum-gray-900 */
-  --spectrum-menu-item-background-color-hover:var(--spectrum-transparent-black-200);
-  --spectrum-menu-item-background-color-down:var(--spectrum-transparent-black-200);
-  --spectrum-menu-item-background-color-key-focus:var(--spectrum-transparent-black-200);
-
```

Comment:
Were all of these `tokens` files affected from a particular version or something? Did you have to use a specific package or something that no longer has these variables?

Reply:
The short answer: These were custom variables that I think were written in because there weren't tokens, but now there are, so I'm (mostly) using those instead.

The longer answer: [It looks like `spectrum-tokens@13.0.0-beta.33`](https://github.com/adobe/spectrum-tokens/releases/tag/%40adobe%2Fspectrum-tokens%4013.0.0-beta.33) adds `--spectrum-menu-item-background-color-default` plus tokens for hover, down, keyboard focus, and disabled states, and they're in the `dist/index.css` from [this PR](https://github.com/adobe/spectrum-css/pull/2797/files#diff-6e8dc6638f2705c29fc830de60cfde41bca79f26502c4bb0607b7e0055786005). Since they're now coming from tokens, I'd rather use that value than the older custom value that was being used before.

Does this (either of these answers 😆 ) answer your questions?